benchmark/intl_stream_benchmark.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:math';
+
+// Import BenchmarkBase class.
+import 'package:benchmark_harness/benchmark_harness.dart';
+
+const int asciiZeroCodeUnit = 2;
+const int listlength = 10000;
+
```

Comment:
Add a comment at the library level describing what 'new' and 'old' mean in this benchmark,
What is it that the benchmarks are showing?